packages/date/src/Date.js

```diff
@@ -109,6 +110,55 @@ const AvDate = ({
     return null;
   };
 
+  const renderMonthElement = ({ month, onMonthSelect, onYearSelect }) => {
+    const { minYear, maxYear } = yearPickerProps;
```

Comment:
Same thing here. Can we not just re-use the `min` and `max` props and derive the current year from there?

I would say if the minDate is Jan 1 then its safe to use that year and the max would be if December 31 is there then the max year would be in there or something to that nature.